packages/create-sitecore-jss/src/templates/nextjs-xmcloud/scripts/config/plugins/edge-platform.ts

```diff
@@ -0,0 +1,29 @@
+import 'dotenv/config';
+import chalk from 'chalk';
+import { constantCase } from 'constant-case';
+import { JssConfig } from 'lib/config';
+import { ConfigPlugin } from '..';
+
+/**
+ * This plugin will set config props used by the Sitecore Edge Platform.
+ */
+class EdgePlatformPlugin implements ConfigPlugin {
+  order = 2;
+
+  async exec(config: JssConfig) {
+    if (config.sitecoreApiKey && config.sitecoreEdgeContextId) {
```

Comment:
_config.sitecoreEdgeContextId_ is not defined yet, use a _process.env[`${constantCase('sitecoreEdgeContextId')}`]_ value

Reply:
fixed